srcpkgs/wayfire/template

```diff
@@ -1,15 +1,15 @@
 # Template file for 'wayfire'
 pkgname=wayfire
-version=0.7.3
-revision=2
+version=0.7.4
```

Comment:
This update seems to be unrelated and should be moved to a separate PR

Reply:
Switched to a revbump instead